app/jobs/hyrax/members_permission_job.rb

```diff
@@ -0,0 +1,23 @@
+module Hyrax
+  # Grants read access for the supplied user for the members attached to a work
+  class MembersPermissionJob < ApplicationJob
```

Comment:
wondering if these might not be better as module mixins, since they aren't queue-able jobs in their own right?